README.md

```diff
@@ -1,21 +1,26 @@
-# gateway-certs-generator
+# Certificate Generator for Slice Gateway
 
-This is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to easy-rsa (typically bundled with OpenVPN).
+The `gateway-certs-generator` is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to `easy-rsa` (typically bundled with OpenVPN).
 
-easy-rsa is a CLI utility to build and manage a PKI CA. In layman's terms, this means to create a root certificate authority, and request and sign certificates including intermediate CAs and certificate revocation lists (CRL).
+To create and maintain a PKI CA, use the CLI tool `easy-rsa`. Using the tool create a root certificate authority, and request and sign certificates including intermediate CAs and certificate revocation lists (CRL).
```

Comment:
A comma should be there after `Using the tool`

Reply:
done